src/client/components/multi-form-editor/landing-page.js

```diff
@@ -1,36 +1,224 @@
 const h = require('react-hyperscript');
 const { Link } = require('react-router-dom');
 const React = require('react');
+const ReactDom = require('react-dom');
+const { makeClassList } = require('../../../util');
+const Promise = require('bluebird');
+const Linkout = require('../document-linkout');
+const anime = require('animejs');
 
 class LandingPage extends React.Component {
+  constructor( props ){
+    super( props );
+
+    this.state = {
+      textAreaEnabled: false,
+      submitting: false,
+      textAreaAnimating: false,
+      textAreaHeight: '20em'
+    };
+  }
+
+  createDoc(){
+    let text = ReactDom.findDOMNode(this).querySelector('.landing-page-text').value;
+
+    let makeRequest = () => fetch('/api/document', {
```

Comment:
@d2fong @maxkfranz  are we using brokers? I would argue that having direct calls to endpoints in components is long-term not too maintainable. Opinions?

Reply:
I think I just moved this code segment from old ```document-filler.js``` page (https://github.com/PathwayCommons/factoid/blob/alpha/src/client/components/document-filler.js#L27)

Reply:
Thanks @metincansiper, let's see what @d2fong @maxkfranz think

Reply:
I think we use direct calls in the code base so I assume this is fine...